src/oscar/templates/oscar/catalogue/category.html

```diff
@@ -3,11 +3,11 @@
 {% load i18n %}
 
 {% block title %}
-    {{ category.name }} | {{ block.super }}
+    {{ category.get_meta_title }} | {{ block.super }}
 {% endblock %}
 
 {% block description %}
-    {{ category.description|default:""|striptags }}
+    {{ category.get_meta_description|striptags }}
```

Comment:
The tag stripping should allready be done in the method to make the result of the method to make sense at all times.